File: src/net/BukkitPE/level/particle/EntityFlameParticle.java
package net.BukkitPE.level.particle;

import net.BukkitPE.math.Vector3;

/**
 * Created on 2015/11/21 by xtypr.
 * Package net.BukkitPE.level.particle in project BukkitPE .
 */
public class EntityFlameParticle extends GenericParticle {
    public EntityFlameParticle(Vector3 pos) {
        super(pos, Particle.TYPE_MOB_FLAME);
    }
}

File: src/BukkitPE/level/particle/EntityFlameParticle.php
<?php
namespace BukkitPE\level\particle;

use BukkitPE\math\Vector3;

class EntityFlameParticle extends GenericParticle{
	public function __construct(Vector3 $pos){
		parent::__construct($pos, Particle::TYPE_MOB_FLAME);
	}
}
